data/language/en-GB.txt

```diff
@@ -3755,6 +3755,8 @@ STR_6304    :Open scenery picker
 STR_6305    :Multithreading
 STR_6306    :{SMALLFONT}{BLACK}Experimental option to use multiple threads to render, may cause instability.
 STR_6307    :Colour scheme: {BLACK}{STRINGID}
+STR_6308    :Trackpad Mode
```

Comment:
I think the label 'Trackpad Mode' has other, more involved associations for people. Might be best to stick to 'Disable scrolling'.